kathy-ems/what2wear
# Updates Data model

#### What's this PR do?
- Updates the data model for better readability and grouping of like items in prep for putting into a sql database. 
- Removes the server side launching of an ejs web page.
  #### What are the important parts of the code?
  The reorg of the data in mainController.
  #### How should this be tested by the reviewer?
  Pull down the branch and run the code to ensure all functionality works and data shows properly.
  #### Is any other information necessary to understand this?
  n/a
  #### What are the relevant tickets? (Please add `closes`, `refs`, etc)
  closes #26 
  #### Screenshots (if appropriate)
  #### Documentation / Release notes updated?
  None to update
